packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -166,6 +169,28 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ```
 /// {@end-tool}
 ///
+/// {@tool sample}
+///
+/// This example of a red [Container] wrapping a [GestureDetector] widget. On
+/// this example the since the [GestureDetector] does not have a child it
```

Comment:
"Since the [GestureDetector] does not have a child it takes on the size of its parent making the entire area of the surrounding [Container] clickable."